### PR TITLE
Fix bug 1207615: added deadline and priority fields to the resource.

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz Hi, could you review? :)

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -687,6 +690,9 @@ class Resource(models.Model):
     format = models.CharField(
         "Format", max_length=20, blank=True, choices=FORMAT_CHOICES)
 
+    deadline = models.DateField(blank=True, null=True)
+    priority = models.IntegerField(choices=((i, i) for i in xrange(1, 6)), default=1)
```
Comment (mathjazz) [inline pontoon/base/models.py]: We should add some meaning ful names, e.g.:

1. Lowest
2. Low
3. Normal
4. High
5. Critical

And default should be 3.

**Comment by mathjazz** [comment]
Nice work!

Dashboards seem broken with new columns added and CSS changes. 

I suggest we leave views, templates, static files untouched until we fix bugs [1203543](https://bugzilla.mozilla.org/show_bug.cgi?id=1203543) and [1203788](https://bugzilla.mozilla.org/show_bug.cgi?id=1203788). So changes to the model + migration should be enough for now.

**Comment by jotes** [comment]
@mathjazz done ;-)

**Comment by mathjazz** [comment]
Great work! r+

**Pull request merged**